### PR TITLE
azurerm 6 upgrade

### PR DESCRIPTION
**Pull request by SINIKI**
## Description
</br>
 Brief description of the requested change.

## Checklist
- [ ] I have read the instructions mentioned in [Contribute to Code](/Contributing.md).
- [ ] I have read and understood the criteria described under [submitting changes](/Contributing.md#submitting-changes). 
- [ ] The title of the PR clearly describes the intent of the PR.
- [ ] This PR does not introduce any breaking changes to the code.

### CODE REVIEW

File: src/AzSK/Framework/Core/FixControl/Services/StorageFix.ps1 [hunk 1]
```diff
@@ -33,23 +33,23 @@ class StorageFix: FixServicesBase
...
-
-	[MessageData[]] EnableFileEncryption([PSObject] $parameters)
-    {
-		[MessageData[]] $detailedLogs = @();
-		$detailedLogs += [MessageData]::new("Enabling encryption on file service of storage [$($this.ResourceName)]...");
-		Set-AzureRmStorageAccount -Name $this.ResourceName -ResourceGroupName $this.ResourceGroupName -StorageEncryption -EnableEncryptionService File
-		$detailedLogs += [MessageData]::new("Encryption on file service is enabled on storage [$($this.ResourceName)]");
-		return $detailedLogs;
-    }
+	# [MessageData[]] EnableBlobEncryption([PSObject] $parameters)
```
[the remark this answers is not in the document]
Comment (ShrutiBhambhani) [reply src/AzSK/Framework/Core/FixControl/Services/StorageFix.ps1]: By default, data is encrypted for Azure Blobs, Tables, Files and Queues. We have removed the corresponding storage controls but fixscript methods are not removed (no need to mention it's not being called from anywhere)

[the remark this answers is not in the document]
Comment (VishalHaibatpure) [reply src/AzSK/Framework/Core/FixControl/Services/StorageFix.ps1]: Remove code

File: src/AzSK/Framework/Configurations/SVT/Services/KeyVault.json [hunk 2]
```diff
@@ -81,7 +81,7 @@
                          "Automated":  "Yes",
                          "MethodName":  "CheckAdvancedAccessPolicies",
                          "Rationale":  "Advanced access policy allows Azure services (Azure Resource Manager, Virtaul Machine, Disk Encryption etc.) to seamlessly access Key Vault. To avoid unintentional access to Key Vault from Azure services, advanced access policies must be configured only as required.",
-                         "Recommendation":  "Remove any advanced policies that are not required using the command: Remove-AzureRmKeyVaultAccessPolicy -VaultName '{VaultName}' -ResourceGroupName '{ResourceGroupName}' -EnabledForDeployment -EnabledForTemplateDeployment -EnabledForDiskEncryption. Refer: https://docs.microsoft.com/en-us/powershell/module/azurerm.keyvault/remove-azurermkeyvaultaccesspolicy?view=azurermps-3.8.0 ",
+                         "Recommendation":  "Remove any advanced policies that are not required using the command: https://docs.microsoft.com/en-us/powershell/module/azurerm.keyvault/remove-azurermkeyvaultaccesspolicy",
```
Comment (VishalHaibatpure) [inline src/AzSK/Framework/Configurations/SVT/Services/KeyVault.json]: Add command

File: src/AzSK/Framework/Configurations/ContinuousAssurance/Continuous_Assurance_Runbook.ps1 [hunk 3]
```diff
@@ -222,13 +222,28 @@ try
 		Write-Output("RB: Started runbook execution...")
 		
 		$appId = $RunAsConnection.ApplicationId 
-        Write-Output ("RB: Logging in to Azure for appId: [$appId]")
-		Add-AzureRmAccount `
+		Write-Output ("RB: Logging in to Azure for appId: [$appId]")
+		if($Null -ne (Get-Command -Name Connect-AzureRmAccount -ErrorAction SilentlyContinue))
```
Comment (sbyna-msft) [inline src/AzSK/Framework/Configurations/ContinuousAssurance/Continuous_Assurance_Runbook.ps1]: try to merge the commands

**Pull request merged**